themes/digital.gov/src/scss/new/_communities.scss

```
@@ -9,6 +9,23 @@
   }
 }
 
+.communities__home {
+  @include u-flex("row", "wrap");
+  display: flex;
+  justify-content: center;
+  padding-top: units(2);
+  &-title {
+    margin-bottom: units(-5);
+    margin-top: units(5);
+  }
+}
```

Empty line before selector for readability.

```suggestion
  padding-top: units(2);
  
  &-title {
    margin-bottom: units(-5);
    margin-top: units(5);
  }
}
```